content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,191 @@
+## Set Up a Proxy to Help Protect Your Validator
 
-4. Make the following configuration changes to your validator:
-    * Copy the `[ips_fixed]` list and paste it under `[ips]`. These fields should contain only the IP addresses and ports of the public-facing rippled(s). The validator connects to only these peers.
-    * Change `[peer_private]` to `1` to prevent its IP address from being forwarded.
+To help protect a validator from [DDoS](https://en.wikipedia.org/wiki/Denial-of-service_attack) attacks, set up one or more stock `rippled` servers as proxies between the validator and inbound traffic. ***TODO: Stated correctly? I think inbound traffic is clearer than "outside network" - but is it accurate? Do we have any recommendations around the minimum number of proxies to set up? Is the primary purpose of providing more than one proxy that if one goes down, you don't lose your connection to the network? Also curious about how the validator's "view" of the network changes when proxies are in place -- what does inbound traffic to a proxy-less validator look like vs inbound traffic to a validator with two proxies, for example? I don't quite understand how the views could differ, if at all.***
 
-5. Configure the validator host machine's firewall to only accept inbound connections from its public-facing rippled(s).
+1. [Enable validation](#enable-validation-on-your-rippled-server) on your `rippled` server.
 
-Remember to restart `rippled` for config changes to take effect.
+2. Set up stock `rippled` servers. For more information, see [Install rippled](install-rippled.html).
 
-Take care not to publish the IP address of your validator.
+3. Configure your validator and stock `rippled` servers to run in a [cluster](cluster-rippled-servers.html).
 
+4. In your validator's `rippled.cfg` file:
 
-## Domain Verification
+    1. Copy the `[ips_fixed]` IP address list you defined in the previous cluster configuration step and paste it under `[ips]`.
 
-Network participants are unlikely to trust validators without knowing who is operating them. To address this concern, validator operators can associate their validator with a web domain that they control.
+        The `[ips_fixed]` and `[ips]` values must be identical and contain only the IP addresses and ports of the stock `rippled` servers you clustered with your validator. This ensures that your validator connects to only the stock `rippled` servers that you control.
 
-1. Find your validator public key by running the following on the validator server:
+    2. Set `[peer_private]` to `1` to prevent your validator's IP address from being forwarded. For more information, see [Private Peers](peer-protocol.html#private-peers).
 
+        **Warning:** Be sure that you don't publish your validator's IP address via any other channels as well.
+
+5. Configure your validator host machine's firewall to accept inbound connections from only the stock `rippled` servers you clustered it with.
```

Comment:
Oh, great catch! 🙏 

Just to talk this through - is this an accurate picture of the firewall settings for the validator:

- Inbound traffic: Only from IP addresses of the stock servers in the cluster. 
- Outbound traffic: Only to the IP addresses of the stock servers in the cluster and to https://vl.ripple.com through port 443. Because even if the validator is getting all other info it needs to participate in consensus from the stock servers in the cluster, the one other thing the validator needs that the stock servers can't provide is the UNL from https://vl.ripple.com? And this is a silly question - but does inbound traffic also need to be open to https://vl.ripple.com for the list of validators to be returned to the validator?

Reply:
Inbound traffic: yes, correct.
Outbund traffic: yes, to the stock servers and to vl.

No need for inbound traffic from VL. vl.ripple.com won't be reaching out and connecting to the validator. Only the reverse.